src/js/components/AppDebugInfoComponent.jsx

```diff
@@ -56,9 +57,13 @@ var AppDebugInfoComponent = React.createClass({
         <dt>Host</dt>
         <dd>{lastTaskFailure.host}</dd>
         <dt>Timestamp</dt>
-        <dd>{lastTaskFailure.timestamp}</dd>
+        <dd>
+          <span>{lastTaskFailure.timestamp}</span> ({new Moment(lastTaskFailure.timestamp).fromNow()})
```

Comment:
Could we extract this to its own variable, so to avoid logic inside JSX?

```
var timestamp = new Moment(lastTaskFailure.timestamp).fromNow();

// ....
<span>{lastTaskFailure.timestamp}</span> ({timestamp})
```

same for L#65